ci_pod/create_ci_pod.sh

```diff
@@ -0,0 +1,60 @@
+
+## SECRETS
+
+# flask
+AWS_DEFAULT_REGION=""
+ENDPOINT_URL=""
+AWS_ACCESS_KEY_ID=""
+AWS_SECRET_ACCESS_KEY=""
+BUCKET=""
+# Add separate ip for ci pod
+POD_CI_IP=""
+BASTION_IP=""
+
+# Change in nginx/Dockerfile and jupyter
+JUPYTERLAB_VERSION=""
+MAINTAINER=""
+USER=""
+TOKEN=""
+
+# MUST replace MANUALLY in nginx/ibm_conf.conf
+JUPYTERLAB_URL=""
+RUN_ARTIFACTS_URL=""
+GRAFANA_URL=""
+KIBANA_URL=""
+ELASTICSEARCH_URL=""
+INTERNAL_URL=""
+
+CI_POD_PATH=""
+
+mkdir -p $CI_POD_PATH/elastic/
+mkdir -p $CI_POD_PATH/grafana/
+mkdir -p $CI_POD_PATH/windows/
+mkdir -p $CI_POD_PATH/jupyterLab/
+
+# chmod solved ci_pod degraded
+chmod 775 -R $CI_POD_PATH/elastic/
+chmod 775 -R $CI_POD_PATH/grafana/
+chmod 775 -R $CI_POD_PATH/windows/
+chmod 775 -R $CI_POD_PATH/jupyterLab/
+
+
+cd $CI_POD_PATH/ci_pod
```

Comment:
In all cases, $CI_POD_PATH *must* be quoted, as it's provided by the user.

Reply:
Fixed